radix-engine/src/blueprints/account/package.rs

```diff
@@ -207,6 +209,7 @@ impl AccountNativePackage {
         }
     }
 
+    #[trace_resources(log=export_name)]
```

Comment:
Can add macro to all `invoke_export` in all packages?

Reply:
Yes, that is good point, it needs to be added to all `invoke_export` functions.